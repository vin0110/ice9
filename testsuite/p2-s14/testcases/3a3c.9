var b: int[10];
var a, a2, a3: int[10][20][30];
b[0] := 0; 
b[1] := 1; 
b[2] := 2; 
b[0] := b[2] + b[1];
a[0][0] := a2[0][1] % a3[2][4];
a[0][0] := a2[0][1] % a3[2][4][2];
a := a2 + a3;
