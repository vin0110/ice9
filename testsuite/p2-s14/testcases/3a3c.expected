line 7: Semantic error: Invalid types in binop
